test/sink/BUILD

```diff
@@ -40,3 +41,11 @@ envoy_cc_test(
         "@envoy//test/test_common:network_utility_lib",
     ],
 )
+
+api_cc_py_proto_library(
+    name = "test_stats_sink_config_proto",
+    srcs = [
+        "test_stats_sink_config.proto",
+    ],
+    visibility = ["//visibility:public"],
```

Comment:
I know we likely didn't pay much attention to visibility here, but we could start looking at this going forward.

Do we need public visibility here? Or can this be made visible just to nighthawk?